### PR TITLE
feat(sace-list-item): fix copact color

### PR DESCRIPTION
**Pull request by torgeadelin**
# Description

Small css fixes for compact mode / size 32 of the SpaceListItem component.
It also includes font colour correction for default vs activity states.

I have also removed the colour on the Text component. This should inherit the default colour from the theme provider. It makes it hard to override if it has a specific class with colour assigned to it.

### CODE REVIEW

**Review by robstax — APPROVED** [review]

File: src/components/ListItemBase/ListItemBase.style.scss [hunk 1]
```diff
@@ -60,8 +65,13 @@
   }
 
   &[data-size='32'] {
+    & > *,
+    .md-text-wrapper {
+      font-weight: 500;
+      font-size: 0.75rem;
+    }
```
Comment (InteractiveTimmy) [inline src/components/ListItemBase/ListItemBase.style.scss:72]: The usage of the comma here is a bit strange. So, all immediate children as well as any child with the class `.md-text-wrapper`? Is there a better way to write this?

**Review by InteractiveTimmy — APPROVED** [review]
One non-blocking comment.

File: src/components/ListItemBase/ListItemBase.style.scss (hunk 1 above)
Comment (torgeadelin) [reply src/components/ListItemBase/ListItemBase.style.scss:72]: Yeah, you got it! I don't know.
I need the immediate children for cases where we have something like.
```js
<ListItemBase>
  <ListItemBaseSection position="fill">Text here</ListItemBaseSection>
</ListItemBase>

//or

<ListItemBase>
  Text here
</ListItemBase>
```

**Review by InteractiveTimmy — APPROVED** [review]
Thank you for adding that comment. Approved.

**Pull request merged**